cli/src/test/kotlin/de/deutschepost/sdm/cdlib/archive/ArchiveCommandIntegrationTest.kt

```
@@ -174,6 +174,7 @@ class ArchiveCommandIntegrationTest(@Value("\${artifactory-azure-identity-token}
         try {
             repository.delete(releaseName_build)
         } catch (_: Exception) {
+            // Exception is intentionally ignored as the absence of the folder is expected and not critical.
```

To address the issue of an empty catch block, which can make the codebase harder to maintain and debug, a comment has been added to clearly indicate that ignoring the exception is intentional. The comment specifies that the absence of the folder (or any associated exceptions) is expected and does not impact the critical functionality, thus explaining the rationale behind ignoring these exceptions. This clarity helps future maintainers understand why this choice was made, preventing any confusion or misinterpretation of the code's behavior.<br><br>[Related Sonar Issue](http://localhost:9000/issues?open=15156638-4782-4aec-a774-3f852ff7c2ca)<br>[Related Sonar Rule](http://localhost:9000/coding_rules?open=kotlin:S108&rule_key=kotlin:S108)